internal/ent/schema/integration.go

```diff
@@ -73,11 +74,16 @@ func (Integration) Mixin() []ent.Mixin {
 	}
 }
 
+// Interceptors of the Integration
+func (Integration) Interceptors() []ent.Interceptor {
```

Comment:
I won't put the same comment on every one of these but not sure why we need to include the interceptor if it's just empty; we don't include hooks, other types in various schemas if there is no actual hook that's called